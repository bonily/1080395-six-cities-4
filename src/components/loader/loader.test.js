import React from "react";
import renderer from "react-test-renderer";
import Loader from "./loader";


describe(`LoaderSnapTest`, () => {
  it(`Loader component should render loader div`, () => {
    const tree = renderer
      .create(<Loader
      />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
