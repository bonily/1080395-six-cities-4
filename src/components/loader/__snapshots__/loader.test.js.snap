// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderSnapTest Loader component should render loader div 1`] = `
<div>
  <header
    className="header"
  >
    <div
      className="container"
    >
      <div
        className="header__wrapper"
      >
        <div
          className="header__left"
        >
          <a
            className="header__logo-link header__logo-link--active"
            onClick={[Function]}
          >
            <img
              alt="6 cities logo"
              className="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
      </div>
    </div>
  </header>
  <div
    style={
      Object {
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    <img
      alt="Loading"
      height="328"
      src="img/loader.svg"
      width="328"
    />
  </div>
</div>
`;
